daemon/container_operations.go

```diff
@@ -719,6 +719,13 @@ func (daemon *Daemon) connectToNetwork(container *container.Container, idOrName
 		return err
 	}
 
+	if !container.Managed {
```

Comment:
@mavenugo I think this change does not change the current behavior that each endpoint calls `addToCluster` only once. Here when an unmanaged container joins a network, `sb.EnableService` would enumerate every endpoint but only the new endpoint would call `addToCluster`. Is this your concern?

```
+func (sb *sandbox) EnableService() error {
 +	for _, ep := range sb.getConnectedEndpoints() {
 +		if ep.enableService(true) {
 +			if err := ep.addToCluster(); err != nil {
 +				ep.enableService(false)
 +				return fmt.Errorf("could not update state for endpoint %s into cluster: %v", ep.Name(), err)
 +			}
 +		}
 +	}
 +	return nil
 +}
```